packages/paste-core/components/badge/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@twilio-paste/badge",
+  "version": "0.0.0",
+  "category": "data display",
+  "status": "alpha",
+  "description": "TODO update.",
+  "author": "Twilio Inc.",
+  "license": "MIT",
+  "main:dev": "src/index.tsx",
+  "main": "dist/index.js",
+  "module": "dist/index.es.js",
+  "types": "dist/index.d.ts",
+  "sideEffects": false,
+  "publishConfig": {
+    "access": "public"
+  },
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "yarn clean && NODE_ENV=production node build.js && tsc",
+    "build:js": "NODE_ENV=development node build.js",
+    "build:props": "typedoc --tsconfig ./tsconfig.json --json ./dist/prop-types.json",
+    "clean": "rm -rf ./dist",
+    "tsc": "tsc"
+  },
+  "peerDependencies": {
+    "@twilio-paste/box": "^4.1.1",
+    "@twilio-paste/button": "^7.0.5",
+    "@twilio-paste/style-props": "^3.0.6",
```

Comment:
When listing peer dependencies don't forget to follow the repo guidelines on hoisting them https://github.com/twilio-labs/paste#peer-deps

Reply:
@SiTaggart Could you take a look at this again? It looks like either some deps are omitted from the button example in the README.md or it is out of date. Thank you!

Reply:
I think that's all of them. Really need a script fo this because it would require me to manually check the dep tree myself